types/globalsat/LT-20/meta.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "LT-20/LT-20P",
+  "version": "1.0.0",
+  "manufacturer": "GlobalSat",
+  "url": "https://www.globalsat.com.tw/en/product-275227/Solar-Powered-LoRa-GPS-Tracker-for-Cattle-Cow-Livestock-and-Farming-LT-20-LT-20P-Series.html",
+  "description": "LT-20 is a solar LoRaWAN tracker designed for range animals use. LT-20P is a solar LoRaWAN tracker designed for asset tracking purpose.",
```

Comment:
what do you mean by: range animals use?

Reply:
probably free range animals zb Cows etc. Changed it anyways